src/testing/Testing.hpp

```diff
@@ -82,6 +82,23 @@ boost::test_tools::predicate_result equals(const Eigen::MatrixBase<DerivedA> &A,
   return true;
 }
 
+/// equals to be used in tests. Prints both operatorans on failure
+template <typename TA, typename TB>
+boost::test_tools::predicate_result equals(const std::vector<TA> &A,
+                                           const std::vector<TB> &B,
+                                           double                 tolerance = math::NUMERICAL_ZERO_DIFFERENCE)
+{
+  Eigen::MatrixXd MatrixA(A.size(), 1);
+  for (int i = 0; i < A.size(); ++i) {
+    MatrixA(i, 0) = A[i];
+  }
+  Eigen::MatrixXd MatrixB(B.size(), 1);
```

Comment:
I think you could do something like `std::copy(A.begin(), A.end(), MatrixA.data())` or is it not possible?

Reply:
Seems to work. Nice :)